firebase-android-release-dashboard/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import {Container, Typography} from "@material-ui/core";
+import {makeStyles} from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  footer: {
+    padding: theme.spacing(2),
+    margin: "auto",
+    textAlign: "center",
+  },
+}));
+
+/**
+ * A stateless functional component that serves as the Footer of the application
+ *
+ * @return {JSX.Element} A Material UI Container component representing the
+ * footer
+ */
+function Footer() {
+  const classes = useStyles();
+
+  return (
+    <Container component="footer" className={classes.footer}>
+      <Typography variant="subtitle1">
+        Support
```

Comment:
What does this support text do? It doesn't seem to link to anything

Reply:
I forgot about this- I wasn't sure what to put in the footer.

Is there a support contact or link I can refer to in the footer?

Reply:
Not one we'd like to expose publically, unless its a link to the issues tab on firebase-android-sdk. 

It might be better to leave off.

Reply:
Sg, removed in https://github.com/firebase/firebase-release-dashboard/pull/14/commits/1827360a08142ebfc78cc257edc71bd2542f6e97